src/examples/yaml_writer_demo.rb

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env ruby
+
+$LOAD_PATH.unshift(File.expand_path('../../lib', __FILE__))
+
+require "storage/fake_probing.rb"
+require "storage/fake_device_factory.rb"
+require "storage/yaml_writer.rb"
+
+input_file  = ARGV[0] || "fake-devicegraphs.yml"
+output_file = ARGV[1] || "/dev/stdout"
+
+fake_probing = Yast::Storage::FakeProbing.new
+devicegraph = fake_probing.devicegraph
+factory = Yast::Storage::FakeDeviceFactory.new(devicegraph)
+factory.load_yaml_file(input_file)
+
+yaml_writer = Yast::Storage::YamlWriter.new
+yaml_writer.write(devicegraph, output_file)
+
+
```

Comment:
I prefer examples to be able to work with my code. This is not intended as a substitute for full-fledged test suites, but as a tool for working. It is not a good idea to deprive us of everything that makes the ever-growing complexity of our work environment worse all the time. We need something to invoke things directly to see what we are doing while we work on the code.